.github/workflows/release.yml

```
@@ -0,0 +1,261 @@
+    permissions:
+      contents: write
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ needs.create-release.outputs.tag }}
+
+      - name: Generate changelog
+        id: changelog
+        run: |
```

actionlint (1.7.7)

property "determine-version" is not defined in object type {create-release: {outputs: {tag: string; version: string}; result: string}; publish-pypi: {outputs: {}; result: string}}

(expression)